src/Intern.php

```diff
@@ -40,9 +40,15 @@ public function run(Experiment $experiment)
      */
     protected function runControl(Experiment $experiment)
     {
+        $argParams = $experiment->getParams();
+
+        if (!empty($experiment->getControlArguments())) {
```

Comment:
Replacing this `if` block with a null coalesce operator on line 43 might be better.